be/src/fs/fs.h

```diff
@@ -102,7 +102,7 @@ struct DirEntry {
 
 struct FileInfo {
     std::string path;
-    std::optional<int64_t> size;
+    std::optional<uint64_t> size;
```

Comment:
No need to represent file size as `uint64_t`, `int64_t` is big enough and `io::SeekableInputStream` is using `int64_t` so, using `uint64_t` here doesn't make much sense.